packages/react-components/src/types/ChatMessage.ts

```diff
@@ -60,8 +74,9 @@ export interface ChatMessage extends MessageCommon {
   mine?: boolean;
   clientMessageId?: string;
   contentType: MessageContentType;
+  readReceipts?: ReadReceipt[];
+  numParticipants?: number;
```

Comment:
Add a doc comment to this with its purpose
```suggestion
/**
 * Total number of participants in the message thread. This is used to display how many participants out of a total have read a message.
 */
 numParticipants?: number;
```

Suggestion: don't shorten num in the public api `numParticipants` --> `numberOfParticipants` .. though this doesn't convey a lot of meaning -- number of participants of what? 
Consider an alternative name like `messageThreadParticipantCount`

Reply:
Updated